inertia/views.py

```diff
@@ -48,6 +49,12 @@ def render_inertia(request, component_name, props=None, template_name=None):
             "in settings.py or pass template parameter."
         )
 
+    # share custom data if any
+    share_method_path = getattr(settings, "INERTIA_SHARE", None)
```

Comment:
please enable it for default.

Reply:
What would be the use of the default function if it does not share any data and if we need to define a custom one in the settings to share some data ? 
Not sure to understand 🤔

Reply:
I agreed, but its good idea had a auth object on the context ...